doc/releases/changelog-0.41.0.md

```diff
@@ -220,323 +240,494 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
   * Transforms that return non-trivial post-processing functions cannot be converted.
   * Transforms will fail to execute if the transformed quantum function or QNode contains:
     * `qml.cond` with dynamic parameters as predicates.
     * `qml.for_loop` with dynamic parameters for ``start``, ``stop``, or ``step``.
     * `qml.while_loop`.
 
-* The sizes of dynamically shaped arrays can now be updated in a `while_loop` and `for_loop`
-  when capture is enabled.
-  [(#7084)](https://github.com/PennyLaneAI/pennylane/pull/7084)
-  [(#7098)](https://github.com/PennyLaneAI/pennylane/pull/7098/)
-
-* `qml.cond` can return arrays with dynamic shapes.
-  [(#6888)](https://github.com/PennyLaneAI/pennylane/pull/6888/)
-  [(#7080)](https://github.com/PennyLaneAI/pennylane/pull/7080)
-
-* `cond`, `adjoint`, `ctrl`, and the `QNode` can now handle accepting dynamically
-  shaped arrays with the abstract shape matching another argument.
-  [(#7059)](https://github.com/PennyLaneAI/pennylane/pull/7059)
-
-* A new `qml.capture.eval_jaxpr` function has been implemented. This is a variant of `jax.core.eval_jaxpr` that can handle the creation
-  of arrays with dynamic shapes.
-  [(#7052)](https://github.com/PennyLaneAI/pennylane/pull/7052)
-
-* The `qml.transforms.single_qubit_fusion` quantum transform can now be applied with program capture enabled.
-  [(#6945)](https://github.com/PennyLaneAI/pennylane/pull/6945)
-  [(#7020)](https://github.com/PennyLaneAI/pennylane/pull/7020)
-
-* The higher order primitives in program capture can now accept inputs with abstract shapes.
-  [(#6786)](https://github.com/PennyLaneAI/pennylane/pull/6786)
-
-* Execution interpreters and `qml.capture.eval_jaxpr` can now handle jax `pjit` primitives when dynamic shapes are being used.
-  [(#7078)](https://github.com/PennyLaneAI/pennylane/pull/7078)
-  [(#7117)](https://github.com/PennyLaneAI/pennylane/pull/7117)
-
-* The `PlxprInterpreter` classes can now handle creating dynamic arrays via `jnp.ones`, `jnp.zeros`,
-  `jnp.arange`, and `jnp.full`.
-  [#6865)](https://github.com/PennyLaneAI/pennylane/pull/6865)
+* Python control flow (`if/else`, `for`, `while`) is now supported when program capture is enabled.
+  [(#6837)](https://github.com/PennyLaneAI/pennylane/pull/6837)
+  [(#6931)](https://github.com/PennyLaneAI/pennylane/pull/6931)
 
-* Added class `qml.capture.transforms.CommuteControlledInterpreter` that moves commuting gates past control
-  and target qubits of controlled operations when experimental program capture is enabled.
-  It follows the same API as `qml.transforms.commute_controlled`.
-  [(#6946)](https://github.com/PennyLaneAI/pennylane/pull/6946)
+  One of the strengths of program capture is preserving a program's structure, eliminating the need 
+  to unroll control flow operations. In previous releases, this could only be accomplished by using 
+  :func:`pennylane.for_loop`, :func:`pennylane.cond`, and :func:`pennylane.while_loop`. With this release, 
```

Comment:
```suggestion
  to unroll control flow operations. In previous releases, this could only be accomplished by using 
  :func:`qml.for_loop <pennylane.for_loop>`, :func:`qml.cond <pennylane.cond>`, and :func:`qml.while_loop <pennylane.while_loop>`. With this release, 
```